presto-docs/src/main/sphinx/release/release-0.252.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.252
+=============
+
+**Highlights**
+==============
+* Add support for returning partial results.
+* Add automatic query retry functionality for transient failures. 
+* Add support for running base32 encoding ElasticSearch queries that are passed through. 
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix the join operator to prevent hanging when spilling is enabled and the probe side finishes before the hash builder starts
+* Add new spilling strategy to spill when a combination of user, system and recovable memory on a node exceeds ``max_total_memory_per_node``. This can be enabled by setting the system configuration ``experimental.spiller.task-spilling-strategy`` to ``PER_QUERY_MEMORY_LIMIT``.
+* Add support for logging formatted prepared query. This can be enabled by setting session property ``log_formatted_query_enabled`` to ``true``.
+* Add support for returning partial results for the queries by setting ``partial_results_enabled`` session property. Additionally ``partial_results_max_execution_time_multiplier``, ``partial_results_completion_ratio_threshold``  session properties can be set to configure the max execution time multiplier and minimum completion ratio threshold for the queries.
+* Add automatic query retry functionality for transient failures. This can be enabled by setting ``per-query-retry-limit`` to a non-zero integer to indicate the per query retry count.
+* Add support to coordinator endpoint ``/v1/info/state`` to return ``ACTIVE`` when the coordinator is not shutting down and the cluster has the minimum required workers.
+* Add functions :func:`chisquared_cdf` and :func:`inverse_chisquared_cdf`.
+
+Security Changes
+________________
+* Add file based password authenticator plugin. See :doc:`security/password-file`.
```

Comment:
this URI also probably needs fixing. Please generate the HTML pages and check that everything looks correct

Reply:
Did you check if this works ? I don't think that this is correct

Reply:
Yeah, I'm sorry - I missed adding the change to the commit before pushing it.

Reply:
It works now.